CODEOWNERS

```diff
@@ -12,4 +12,4 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-*  @bhs @bogdandrutu @sarahnovotny @SergeyKanzhelev @yurishkuro
+*  @opentelemetry/admins
```

Comment:
is the admins list actually visible publicly?

![image](https://user-images.githubusercontent.com/3523016/71202822-e90c5a00-226a-11ea-94fe-56c687c14cb0.png)

Reply:
We should make it publicly visible IMO.

Reply:
@yurishkuro you have access to update the visibility settings.

Reply:
changed. I assume it's not case sensitive.